src/applications/disability-benefits/all-claims/pages/addDisabilities.js

```diff
@@ -38,7 +51,25 @@ export const uiSchema = {
         },
       ),
       'view:descriptionInfo': {
-        'ui:description': uiDescription,
+        'ui:description': descriptionInfo,
+      },
+    },
+  },
+  // This object only shows up when the user tries to continue without claiming either a rated or new condition
+  'view:newDisabilityErrors': {
+    'view:newOnlyAlert': {
+      'ui:description': newOnlyAlert,
+      'ui:options': {
+        hideIf: formData =>
+          !newConditionsOnly(formData) || hasClaimedConditions(formData),
```

Comment:
Are both of these needed? If they don't have any claimed conditions by this point, it probably doesn't matter whether or not they selected something other than 'new only,' I'd think we just want them to enter a new disability regardless, right?